### ISSUE TITLE
Q: [Restaurant] Direction from a start position to client' tables 

### ISSUE THREAD

**Issue by kochigami** [RoboCupAtHome/RuleBook#342]
Does a robot start at a direction where it can see the client's tables without moving around?

In p. 80 (6.3 Restarant/ 6.3.3 Task/ 1. Start ) says that "After the start signal is given,
the robot may look around to keep an eye on the tables."
I'd like to know whether a robot should search for the tables after start signal is given and before it looks around the tables.

example
![untitled diagram](https://user-images.githubusercontent.com/7259671/28243598-0163d6ba-6a0c-11e7-8941-dce7d728a55c.png)

**Comment by kyordhel** [comment]
We don't know, and probably we won't know until the test itself. The robot
should behave as any human waiter getting familiarised with the restaurant
in which will work.

We cannot guarantee that all the tables and customer tables will be visible
from the entrance or the kitchen, or from wherever the robot start. In
previous competitions we have had bars with corners (L-shaped), tall
tables, glass walls and natural sunlight (Mexico 2012); fancy round shaped
restaurants with the service bar in an island in the centre and the tables
around (Leipzig 2016); cafes with extra tall tables and benches next to a
corridor in a not very well delimited area (Eindhoven 2013); and improvised
ones with no real tables but just waypoints (João Pessoa, 2014).

In conclusion, take the best strategy for your robot as long as it relies
on a natural behaviour.

**Comment by kochigami** [comment]
Thank you very much for your quick response and information of the past.
It really helped me.
